.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  root: true,
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+  ],
+  parserOptions: {
+    project: './tsconfig.json',
+  },
+  ignorePatterns: ['dist/**/*'],
+  rules: {
+    '@typescript-eslint/no-explicit-any': 'error',
```

Comment:
Quite surprised to see this achievable. I guess the functions are simple enough and we're lucky to depend on well-maintained libraries.